cucumber/api/features/support/rest_helpers.rb

```diff
@@ -173,13 +187,27 @@ def create_user login, owner
       @user_index += 1
     end
 
-    return if users[login]
-
     roleid = "cucumber:user:#{login}"
-    Role.create(role_id: roleid).tap do |user|
-      Credentials[role: user] || Credentials.new(role: user).save(raise_on_save_failure: true)
+    create_role(roleid, owner)
+  end
+
+  # Create a regular host, owned by the admin user
+  def create_host login, owner
+    unless login
+      login = USER_NAMES[@host_index]
```

Comment:
why do we need this array? RestHelpers defines this as 
```
USER_NAMES = %w[auto-larry auto-mike auto-norbert auto-otto].freeze
```
so I am trying to understand why we have this and a host_index?

Reply:
we had it for the default username. I removed it.